test/api/controllers/adminController.test.js

```diff
@@ -1,595 +1,510 @@
 var
-  should = require('should'),
-  _ = require('lodash'),
   Promise = require('bluebird'),
+  rewire = require('rewire'),
+  should = require('should'),
   sinon = require('sinon'),
-  params = require('rc')('kuzzle'),
-  Kuzzle = require.main.require('lib/api/Kuzzle'),
-  User = require.main.require('lib/api/core/models/security/user'),
-  Profile = require.main.require('lib/api/core/models/security/profile'),
-  Role = require.main.require('lib/api/core/models/security/role'),
+  AdminController = rewire('../../../lib/api/controllers/adminController'),
```

Comment:
No need to use rewire, we should just load the constructor with `require.main.require`